src/Symfony/Component/HttpClient/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+7.3
+--
+
+* Allow `AmphpHttpClient` with amphp/http-client v5 to use unix socket
```

Comment:
this should be reverted: the change is a bugfix, not a new feature (the PR should target branch